src/components/Grid/Grid.tsx

```diff
@@ -1,15 +1,28 @@
-import Cell from '../Cell/Cell'
-import './Grid.scss'
-import { cellsMockData } from './cellsMockData'
+import { useState } from 'react';
+import Cell from '../Cell/Cell';
+import Modal from '../Modal/Modal';
+import './Grid.scss';
+import { cellsMockData } from './cellsMockData';
 
 const Grid = () => {
-    const cells = cellsMockData.map(cell => <Cell id={cell.id} key={cell.id}/>)
+  const [modal, setModal] = useState<boolean>(false);
 
-    return (
-        <section id='grid'>
-            {cells}
-        </section>
-    )
+  const openModal = () => {
+    setModal(true);
+  };
+
+  const closeModal = () => {
+    setModal(false);
+  };
```

Comment:
could dry this up with a single function `setModal(!modal)`

Reply:
done! just changed this